lib/parse.js

```diff
@@ -2132,14 +2132,20 @@ function parse($TEXT, options) {
                     end: prev()
                 });
             } else {
+                const start = S.token;
+                const symbol = as_symbol(sym_type);
+                const init_value =  is("operator", "=") ?
+                    (next(), expression(false, no_in))
+                    : !no_in && kind === "const" ? croak("Missing initializer in const declaration") : null;
+                const pre = prev();
+                if (sym_type === AST_SymbolConst) {
+                    symbol.init = init_value;
+                }
```

Comment:
currently, AST_Symbol_Const seems would not have any init property. so I add init to the symbol when it is created.

Reply:
The `.init` property should still be there, in the "def". It shouldn't be connected to the symbol, since every symbol will connect to a def and you can retrieve .init from there.